cli/src/account.rs

```diff
@@ -0,0 +1,34 @@
+use clap::ArgMatches;
+use stellar_client::{sync, endpoint::{account, Order}, error::Result, sync::Client};
+use super::utils;
+
+pub fn details<'a>(client: Client, matches: &'a ArgMatches) -> Result<()> {
+    let id = matches.value_of("ID").expect("ID is required");
+    let endpoint = account::Details::new(id);
+    let account = client.request(endpoint)?;
+
+    println!("ID:       {}", account.id());
+    println!("Sequence: {}", account.sequence());
+
+    Ok(())
+}
+
+pub fn transactions<'a>(client: Client, matches: &'a ArgMatches) -> Result<()> {
+    let id = matches.value_of("ID").expect("ID is required");
+    let endpoint = account::Transactions::new(id).order(Order::Desc);
```

Comment:
Do we want limit, order, or cursor to be options?

Reply:
perhaps. Order makes sense, cursor could be a starts-from, and limit is page size (which may not control how many we actually fetch at a time). However, that can be another branch as I think this one contains enough.